pkg/logger/logger.go

```diff
@@ -0,0 +1,9 @@
+package logger
+
+import "sigs.k8s.io/controller-runtime/pkg/log/zap"
+
+var (
+	Options = zap.Options{
+		Development: true,
```

Comment:
Is changing this to `false` handled if that is the intention?

Reply:
not yet but should be made configurable

Reply:
it was already there, so removed this default set-up